run_dir/design/flowcell.html

```diff
@@ -78,11 +78,11 @@
                         title="Threshold: {{ threshold }} million">{% for thousand in lane.get('clustersnb', '').split(',') %}<span class="thousand_group">{{ thousand }}</span>{% end %}
                     </td>
                     <th>%&nbsp;bases&nbsp;>&nbsp;Q30:</th>
+                    {% set run_setup = flowcell.get('run_setup', '').split('x')[-1] %}
+                    {% try %} {% set run_setup = int(run_setup) %} {% except %}{% end %}
```

Comment:
Why don't you set the default of run_setup to '0'  or something ? That way, you could avoid the try/except.